Tools/BuildAndPackage.ps1

```diff
@@ -0,0 +1,92 @@
+# Builds the Google Cloud Tools for PowerShell project, and packages the output so it can be
+# integrated into the Google Cloud SDK. This is really of only use to Googlers doing the
+# official releases of the PowerShell module.
+
+# Bin directory for the Google.PowerShell project.
+$binDir = Join-Path $PSScriptRoot "\..\Google.PowerShell\bin\"
+$binDir = [System.IO.Path]::GetFullPath($binDir)
```

Comment:
`Resolve-Path` seems to require the path actually exist. So if the `$binDir` has already been deleted, `Resolve-Path` will fail.